src/api/app/components/notification_excerpt_component.rb

```diff
@@ -14,7 +14,7 @@ def call
              @notifiable.description.to_s # description can be nil
            when 'Comment'
              helpers.render_without_markdown(@notifiable.body)
-           when 'Report', 'Decision', 'Appeal'
+           when 'Report', 'Decision', 'Appeal', 'DecisionFavoredWithDeleteRequest'
```

Comment:
I added the type to the notification excerpt component so we get the reason for the report in the notification body

Reply:
It's done here https://github.com/openSUSE/open-build-service/pull/16046